src/backend/base/langflow/graph/graph/state_model.py

```diff
@@ -63,6 +63,6 @@ def create_state_model_from_graph(graph: BaseModel) -> type[BaseModel]:
     ]
     fields = {
         camel_to_snake(vertex.id): state_model_getter
-        for vertex, state_model_getter in zip(graph.vertices, state_model_getters)
+        for vertex, state_model_getter in zip(graph.vertices, state_model_getters, strict=False)
```

Comment:
Reading the code, it seems `graph.vertices` and `state_model_getters` could have distinct size. Is it intended ?

Reply:
state_model_getters is built by iterating graph.vertices

Reply:
Yeah. We probably need to add the same checks on the second iteration or just build the fields dict in one go.

Reply:
This part is still highly experimental and requires a lot more testing/inclusion into features